Gemfile

```diff
@@ -22,7 +22,7 @@ gem 'sidekiq-limit_fetch'
 gem "sidekiq-statsd", "0.1.5"
 gem "sinatra", "~> 2.0.0"
 gem "slop", "~> 3.4.5"
-gem "statsd-ruby", "~> 1.3.0"
+gem "statsd-ruby", "~> 1.4.0"
```

Comment:
do we need to manually add this given it is a dependency of `govuk_app_config`

Reply:
Rummager makes some calls to statsd directly (e.g. in the PublishingEventProcessor), so I think it's worth including this an explicit dependency.